# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nicstat/no-sudo.patch
# Copyright (C) 2013 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- nicstat-src-1.92/Makefile.Linux.vanilla	2013-10-14 08:45:02.000000000 +0200
+++ nicstat-src-1.92/Makefile.Linux	2013-10-14 08:46:59.000000000 +0200
@@ -16,8 +16,8 @@
 
 CFLAGS =	$(COPT) -m32
 
-INSTALL =	sudo install -o bin -g bin
-SETUINSTALL =	sudo install -o root -g root -m 4511
+INSTALL =	install -o bin -g bin
+SETUINSTALL =	install -o root -g root -m 4511
 
 #-- Look in nicstat.sh to see if you want to be more specific
 OSTYPE =	RedHat
